doc/running.rst

```diff
@@ -88,6 +91,14 @@ An example batch script for the slurm batch system is given below:
    #SBATCH -t 00:30:00                 # walltime
    #SBATCH -p pbatch                   # queue to use
 
+   # Run this script with 'sbatch <script.sh>'
+
+   # Put any environment activation here, e.g.:
+   # source ../../config/activate_env.sh
+
+   # Put any device selection here, e.g.:
+   # export PYOPENCL_CTX=":"
+
    nnodes=$SLURM_JOB_NUM_NODES
    nproc=$nnodes # 1 rank per node
 
```

Comment:
This bit looks pretty redundant with the script in examples.

How about
```rst
.. literalinclude:: ../examples/lassen.bsub.sh
    :language: bash
```
instead? (Also below)

Reply:
good idea, changed.